src/main/java/duke/Duke.java

```diff
@@ -1,8 +1,17 @@
 package duke;
 
 import java.util.Scanner;
+
+/**
+ * This is Duke who initialises Joe the Java Bot!
+ */
 public class Duke {
 
+    /**
+     * Runs Joe, my java bot!
+     *
+     * @param args Input arguments, will be ignored.
+     */
     public static void main(String[] args) {
```

Comment:
Maybe you could consider abstracting out the behaviour of the bot into a method run()?

Reply:
Think I agree with this, I believe some lines here can be abstracted out into the Joe class.